files/buildspec_tf_plan.yml.tftpl

```diff
@@ -2,7 +2,9 @@ version: 0.2
 
 env:
   variables:
-    TF_VERSION: "1.2.9"
+    TF_VERSION: ""
+    DIRECTORY: ""
+    EXTRA_FILE: ""
```

Comment:
`EXTRA_FILE` vs `EXTRA_FILES` -

Reply:
Is it really an environmental variable or a template variable? IDK how the templating works here.

Reply:
Reading from https://developer.hashicorp.com/terraform/language/functions/templatefile

we don't need to define the environmental variables here - the values will be replaced by `templatefile()`. If we want to use environmental variables instead, then we provide them in aws_codebuild_project.tflint.environment - as far as I understand it.